libindy/src/services/wallet/storage/default/mod.rs

```diff
@@ -176,7 +177,13 @@ impl StorageIterator for SQLiteStorageIterator {
                     None
                 };
                 let tags = if self.options.fetch_tags {
-                    Some(self.tag_retriever.as_mut().unwrap().retrieve(row.get(0))?)
+                    if let Some(ref mut tag_retriever) = &mut self.tag_retriever {
```

Comment:
I have compilation error here. It looks like you use another version of Rust

Reply:
interesting. Yes, the syntax looks a bit strange, maybe it was not possible before. I use 1.26